test/specs/test.apply.claim.journeys.js

```diff
@@ -0,0 +1,53 @@
+import { expect, browser, $ } from "@wdio/globals";
+import * as common from "../utils/common.js";
```

Comment:
lets import the functions we need from files, rather than doing the `import * as` approach. Its generally a bad practice unless theres a specific reason to do so.